ext/ffi/ffi.c

```diff
@@ -514,6 +514,9 @@ static zend_always_inline void zend_ffi_cdata_to_zval(zend_ffi_cdata *cdata, voi
 				if (*(void**)ptr == NULL) {
 					ZVAL_NULL(rv);
 					return;
+				} else if (debug_union) {
+					ZVAL_STR(rv, zend_strpprintf(2 * SIZEOF_SIZE_T + sizeof("0x"), "%p", *(void**)ptr));
```

Comment:
I'd just pass 0 here. Don't think we ever try to be more precise here...

Reply:
Nice simplification. Thanks!